src/databases/PlainText/avtPlainTextFileFormat.C

```diff
@@ -34,6 +34,16 @@
 using     std::string;
 using     std::vector;
 
+string
+TrimLeadingandTrailingWhitespace(string str)
+{
+    auto start = str.find_first_not_of(" \t");
```

Comment:
There is a C function `isspace` that considers `' ', '\t', '\n', '\v', '\f' and '\r'` as whitespace. So I think using `" \t\n\v\f\r"` would be the way to go.

Reply:
I was looking at `isspace`, in fact, that's where I got the list I posted up above. I'll go ahead and add the others as you've suggested, but other than tabs and spaces, the rest are all line breaks, which I thought shouldn't be in the first line anyway. Should running in to any of those characters while reading a line result in an error?

Reply:
Good point! You should leave it as is. I approved your pull request.